sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserService.java

```diff
@@ -715,7 +715,10 @@ public Predicate createCurrentUserJurisdictionFilter(CriteriaBuilder cb, From<?,
 		if (hasRight(UserRight.SEE_PERSONAL_DATA_OUTSIDE_JURISDICTION)) {
 			return cb.conjunction();
 		}
+		return createCurrentUserJurisdictionFilterForTasks(cb, from);
+	}
 
+	public Predicate createCurrentUserJurisdictionFilterForTasks(CriteriaBuilder cb, From<?, User> from) {
```

Comment:
I'd like these two methods to be refactored, because the content of this method doesn't really have anything to do with tasks; my suggestion would be to call this method "createCurrentUserJurisdictionFilter", and remove the method above because it's only used in one place anyway; instead, move its content to the place where it's currently used

Reply:
changed